pkg/controller/kibana/kibana_controller.go

```diff
@@ -0,0 +1,104 @@
+package kibana
```

Comment:
nit but since it lives in `controller/kibana` you could just call it `controller.go` or `reconcile.go`

Reply:
I agree, but this is how the `operator-sdk` names the controllers. Anyway, I fixed that.